service/terraform-plugin/src/test/java/com/solace/maas/ep/event/management/agent/plugin/terraform/TerraformCommandIT.java

```diff
@@ -72,6 +71,64 @@ public class TerraformCommandIT {
     @Autowired
     private ResourceLoader resourceLoader;
 
+    private static Command generateCommand(String tfCommand, String body) {
+        return generateCommand(tfCommand, body, false);
+    }
+
+    private static Command generateCommand(String tfCommand, String body, Boolean ignoreResult) {
+        return generateCommand(tfCommand, body, ignoreResult,
+                Map.of("Content-Type", "application/hcl",
+                        "Content-Encoding", "base64"));
+    }
+
+    private static Command generateCommand(String tfCommand, String body, Boolean ignoreResult, Map<String, Object> parameters) {
+        return Command.builder()
+                .body(Optional.ofNullable(body)
+                        .map(b -> Base64.getEncoder().encodeToString(b.getBytes(UTF_8)))
+                        .orElse(""))
+                .command(tfCommand)
+                .commandType(CommandType.terraform)
+                .ignoreResult(ignoreResult)
+                .parameters(parameters)
+                .build();
+    }
+
+    private static CommandRequest generateCommandRequest(Command commandRequest) {
+        return generateCommandRequest(List.of(commandRequest), false);
+    }
+
+    private static CommandRequest generateCommandRequest(List<Command> commandRequests, boolean exitOnFailure) {
+        return CommandRequest.builder()
+                .commandBundles(List.of(
+                        CommandBundle.builder()
+                                .executionType(ExecutionType.serial)
+                                .exitOnFailure(exitOnFailure)
+                                .commands(commandRequests)
+                                .build()))
+                .commandCorrelationId("234")
+                .context("app123")
+                .serviceId("ms1234")
+                .build();
+    }
```

Comment:
I am finding it a little difficult to understand the diff here, if I am not wrong, its the exact same code but you pulled it on to the top? @mynecker